src/main/java/com/kodilla/ecommercee/controller/ProductController.java

```diff
@@ -0,0 +1,40 @@
+package com.kodilla.ecommercee.controller;
+
+import com.kodilla.ecommercee.domain.dto.ProductDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("v1/shop/products")
+@RequiredArgsConstructor
+public class ProductController {
+
+    @GetMapping
+    public List<ProductDto> getAllProducts() {
```

Comment:
wystarczy getProducts()

Reply:
Nomenklatura... Znowu bliżej mi do nazewnictwa CrudRepository

	/**
	 * Returns all instances of the type.
	 *
	 * @return all entities
	 */
	Iterable<T> findAll();

Reply:
Mi odwrotnie :) myślę że do ustalenia na forum, ja się akurat przyzwyczaiłam juz do tego jak robiliśmy w TaskApplication.